mockServer/server.ts

```diff
@@ -1,16 +1,16 @@
 import cors from "cors";
 import express, { Express } from "express";
 import gardensRouter from "./routes/gardens";
-import tmpRouter from "./routes/tmp";
+import rulesRouter from "./routes/rules";
 
 const app: Express = express();
 
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.use("/api/v1/tmp", tmpRouter);
 app.use("/api/v1/gardens", gardensRouter);
+app.use("/api/v1/rules", rulesRouter);
 
 app.listen(3001, () => {
```

Comment:
Added currently using api mock data, so please try to use mock server for dev and add mock data when adding a new api. (need to change the env var for api base url to http://localhost:3001/api/v1)